<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Frame</class>
 <widget class="QWidget" name="Music_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>900</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 159, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Lrc_Btn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>520</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/lrc</normaloff>:/play/lrc</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="Volume_Process">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>530</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; 
     border-radius: 3px;
 }</string>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Volume_Btn">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>525</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/volume_noraml</normaloff>:/play/volume_noraml</iconset>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_Btn">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>520</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/sub/return</normaloff>:/sub/return</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="current_time">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_Btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>515</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/next</normaloff>:/play/next</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Mode_Btn">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>520</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/list_cycle</normaloff>:/play/list_cycle</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Pre_Btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>515</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/pre</normaloff>:/play/pre</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="totle_time">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>525</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="Music_process">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>365</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 0; 
     border-radius: 3px;
 }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="Switch_wdiget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>900</width>
     <height>465</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="song_content"/>
   <widget class="QWidget" name="Music_Gedan">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>145</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="splitterLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>900</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="List_Btn">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>List</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_Btn">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Zhubo_Btn">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Zhubo</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Paih_Btn">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Paihangb</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Download_Btn">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="New_music_Btn">
   <property name="geometry">
    <rect>
     <x>719</x>
     <y>5</y>
     <width>72</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Play_Btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/img.qrc">
     <normaloff>:/play/player</normaloff>:/play/player</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../res/img.qrc"/>
 </resources>
 <connections/>
</ui>
